docs/tutorials/tutorial_experience.ipynb

```diff
@@ -0,0 +1,859 @@
+{
```

Comment:
<p>This tutorial should precede the one on observable characteristics. This here is more basic, isnt it? Even if not, if we do not mention other tutorials in a particular order and just link to them, we can change the order without problems. I think this is desirable. It automatically produces self-contained sections.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/375/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>